docs/root/configuration/http_filters/ext_authz_filter.rst

```diff
@@ -13,7 +13,8 @@ also possible. This is explained in more details at :ref:`HTTP filter <envoy_api
 
 .. tip::
   It is recommended that this filter is configured first in the filter chain so that requests are
-  authorized prior to the rest of filters processing the request.
+  authorized prior to the rest of filters processing the request. This avoids wasting processing
```

Comment:
TBH, I might just delete this entire tip, since IMO it's not always true. For example, for edge traffic you generally want to rate limit before auth so you protect the auth server. cc @gsagula

Reply:
+1 for deleting.